### ISSUE TITLE
G2.1: Adopt new ServiceX frontend version 3.0

### ISSUE THREAD

**Issue by matthewfeickert** [iris-hep/analysis-systems-deliverables#1]
https://github.com/ssl-hep/ServiceX_frontend

* [ServiceX API 3.0 Design notes](https://codimd.web.cern.ch/f9t1I7ZHRkm98bDmLFHBzg) from 2024-02-15
* https://github.com/ssl-hep/ServiceX_frontend/milestone/9
* https://github.com/ssl-hep/ServiceX_frontend/issues/347
* https://github.com/ssl-hep/ServiceX_frontend/issues/363

**Comment by matthewfeickert** [comment]
From the [2024-06-10 Executive Board meeting](https://docs.google.com/document/d/1NRhrigUuei0PHxg8pgH0QaJ3pxbh6vFIh2M0F57_oJQ/edit), we're going to need to have a more definite roadmap here. @bbockelm suggested a strategy meeting, which is realistically needed.

**Comment by matthewfeickert** [comment]
The [ServiceX client `v3.0.0` release](https://github.com/ssl-hep/ServiceX_frontend/releases/tag/3.0.0) is now out.

**Comment by matthewfeickert** [comment]
@ponyisi @gordonwatts @BenGalewsky Can you comment on the overall integration status of the v3.0.0 release of the client into existing workflows, like the AGC demonstrators or public demos?

**Comment by ponyisi** [comment]
Hi @matthewfeickert - I would also ping @alexander-held and @oshadura - what I do know is that we have a PR (pending a few small cleanups) for integrating this in the AGC repo with ATLAS and CMS examples. But I think that is more or less what has been done from the ServiceX side.

**Comment by matthewfeickert** [comment]
I'm also noticing that in the [ServiceX client docs for FuncADL Query Type](https://servicex-frontend.readthedocs.io/en/3.0.0/query_types.html#funcadl-query-type) that the full user docs are listed as @gordonwatts's https://github.com/gordonwatts/xaod_usage [Jupyter Book](https://gordonwatts.github.io/xaod_usage/). However, that Jupyter Book isn't using the `v3.0.x` API, so this will be important to update for us to be able to say that the `v3.0.x` client has been adopted.

Cross reference issues:
* https://github.com/ssl-hep/ServiceX_frontend/issues/493
* https://github.com/gordonwatts/xaod_usage/issues/23

**Comment by gordonwatts** [comment]
3.0 is in AGC, now, right? @alexander-held

**Comment by alexander-held** [comment]
Yes, credit to @ponyisi: https://github.com/iris-hep/analysis-grand-challenge/pull/225.

**Comment by gordonwatts** [comment]
Awesome! Then lets close this.

**Comment by matthewfeickert** [comment]
Yeah, given @gordonwatts's feedback on https://github.com/ssl-hep/ServiceX_frontend/issues/493#issuecomment-2675464092 I think this is fully resolved. 👍 

Nice job, ServiceX team!
